docs/managed-updates.md

```diff
@@ -0,0 +1,275 @@
+# Managed Updates (Alpha)
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+RFC 2119.
+
+## Overview
+
+This is a guide to integrate with Managed Updates.
+
+Managed updates uses Kubernetes Application Lifecycle Management (KALM)
+framework.
+
+## Prerequisites
+
+We're using a pre-release of the toolchain. Setup the following environment
+variable for your shell:
+
+```shell
+export MARKETPLACE_TOOLS_TAG=0.8.0-alpha02
+```
+
+### Install mpdev
+
+See [this doc](tool-prerequisites.md).
+
+### GCS bucket
+
+Create a GCS bucket to house the published version metadata. This SHOULD
+be a bucket in the same project you're publishing images and SHOULD be
+dedicated to this process.
+
+The bucket MUST be set to be publicly _readable and listable_. See the
```

Comment:
Is this a typo?
_readable
listable_

Reply:
Revised this paragraph to include a direct CLI command instead.